patches/display-language.diff

```diff
@@ -79,7 +79,7 @@ Index: code-server/lib/vscode/src/vs/code/browser/workbench/workbench.html
 +		} catch (error) { /* Probably fine. */ }
  		Object.keys(self.webPackagePaths).map(function (key, index) {
  			self.webPackagePaths[key] = new URL(
- 				`{{VS_BASE}}/static/remote/web/node_modules/${key}/${self.webPackagePaths[key]}`,
+ 				`{{VS_BASE}}/static/node_modules/${key}/${self.webPackagePaths[key]}`,
```

Comment:
Does this need to be changed in my upstream PR?

Reply:
Nope, this is not part of the changes this patch applies, it is just part of the surrounding context.  So this change is from an earlier patch.